tutorials/multimodal/NeVA Tutorial.ipynb

```diff
@@ -0,0 +1,253 @@
+    "1. Training a NeVA model\n",
+    "2. Performing inference with the trained model\n",
+    "\n",
+    "## Datasets\n",
+    "\n",
+    "After downloading all below datasets for pretraining and instruction tuning, your dataset directory should look something similar to:\n",
+    "\n",
+    "```\n",
+    "LLaVA-Pretrain-LCS-558K\n",
+    "├── blip_laion_cc_sbu_558k.json\n",
+    "├── images\n",
+    "LLaVA-Instruct-mixture\n",
+    "├── llava_v1_5_mix665k.json\n",
+    "└── images\n",
+    "    └── ...\n",
+    "```\n",
+    "\n",
+    "### Pre-Training Dataset\n",
+    "\n",
+    "The pre-training dataset is open-sourced from the LLaVA implementation and can be downloaded [here](https://huggingface.co/datasets/liuhaotian/LLaVA-Pretrain). The dataset consists of a 558K subset of the LAION-CC-SBU dataset with BLIP captions. \n",
+    "\n",
+    "The associated images for pretraining can be downloaded via HuggingFace [here](https://huggingface.co/datasets/liuhaotian/LLaVA-Pretrain/blob/main/images.zip).\n",
+    "\n",
+    "### Instruction Tuning Dataset\n",
+    "\n",
+    "The instruction tuning annotations are sourced from the LLaVA implementation and are available [here](https://huggingface.co/datasets/liuhaotian/LLaVA-Instruct-150K/blob/main/llava_v1_5_mix665k.json).\n",
+    "\n",
+    "The associated images for the mixture instruction tuning annotations can be found [here](https://github.com/haotian-liu/LLaVA?tab=readme-ov-file#visual-instruction-tuning). After extracting, the data should be formatted as follows:\n",
+    "\n",
+    "```\n",
+    "    images\n",
+    "      ├── coco\n",
+    "      │    └── train2017\n",
+    "      ├── gqa\n",
+    "      │    └── images\n",
+    "      ├── ocr_vqa\n",
+    "      │    └── images\n",
+    "      ├── textvqa\n",
+    "      │    └── train_images\n",
+    "      └── vg\n",
+    "           ├── VG_100K\n",
+    "           └── VG_100K_2\n",
+    "```\n",
+    "\n",
+    "## Training\n",
+    "\n",
+    "\n",
+    "### Feature Alignment Pre-Training\n",
+    "\n",
+    "We provide a set of scripts for pre-training and fine-tuning which can be kicked off with CLI flags defining specified arguments. \n",
+    "\n",
+    "An example of a pre-training script:\n",
+    "\n",
+    "```\n",
+    "torchrun --nproc_per_node=4 /opt/NeMo/examples/multimodal/multimodal_llm/neva/neva_pretrain.py \\\n",
+    " ++cluster_type=BCP \\\n",
+    " trainer.precision=bf16 \\\n",
+    " model.megatron_amp_O2=True \\\n",
+    " trainer.num_nodes=1 \\\n",
+    " trainer.devices=4 \\\n",
+    " trainer.val_check_interval=1000 \\\n",
+    " trainer.limit_val_batches=5 \\\n",
+    " trainer.log_every_n_steps=1 \\\n",
+    " trainer.max_steps=1000 \\\n",
+    " model.micro_batch_size=1 \\\n",
+    " model.global_batch_size=2 \\\n",
+    " model.tensor_model_parallel_size=4 \\\n",
+    " model.pipeline_model_parallel_size=1 \\\n",
+    " model.mcore_gpt=False \\\n",
```

Comment:
should mcore_gpt and te be set to True?

Reply:
This example is based off of the config files provided in `examples/multimodal/multimodal_llm/neva/conf/neva_config.yaml`, which have `mcore_gpt` and `transformer_engine` set to `False` [here](https://github.com/NVIDIA/NeMo/blob/aeb979987f64dd341885f0744cbe92cb881ffa76/examples/multimodal/multimodal_llm/neva/conf/neva_config.yaml#L88) and [here](https://github.com/NVIDIA/NeMo/blob/aeb979987f64dd341885f0744cbe92cb881ffa76/examples/multimodal/multimodal_llm/neva/conf/neva_config.yaml#L151). So I believe they should be set to `false`. @yaoyu-33 any input here?

Reply:
Good catch. They should both be set to True now. I will update the yaml file. @PannuMuthu  plz update to True